packages/fuselage/src/components/Card/Card.styles.scss

```
@@ -83,4 +83,16 @@ $card-hero-padding: lengths.padding(28);
     display: flex;
     align-items: center;
   }
+
+  &__controls {
+    flex-wrap: wrap;
+  }
+
+  &__horizontal &__controls {
+    flex-wrap: nowrap;
+
+    &--wrap {
+      flex-wrap: wrap;
+    }
+  }
```

```suggestion
  &__horizontal &__controls {
    flex-wrap: wrap;

    @include on-breakpoint(sm) {
      flex-wrap: nowrap;
    }
  }
```
The mixin in question, also I've updated your logic to follow a 'mobile first' flow.